sqlalchemy_bigquery/base.py

```diff
@@ -261,6 +261,7 @@ def _known_tables(self):
             if isinstance(from_, Table):
                 known_tables.add(from_.name)
             elif isinstance(from_, CTE):
+                known_tables.add(from_.name)
```

Comment:
Any possibility you could add a test that was failing before this change? Or at least provide a code example so that one of the package maintainers could add such a test.

Reply:
Unit tests added